src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -248,6 +254,39 @@ std::string parseSubscription(ConnectionInfo* ciP, ParseData* parseDataP, JsonDe
     destination->throttling.seconds = 0;
   }
 
+
+  // attrsFormat field
+  if (document.HasMember("attrsFormat"))
+  {
+    const Value& attrsFormat = document["attrsFormat"];
+
+    if (!attrsFormat.IsString())
+    {
+      OrionError oe(SccBadRequest, "attrsFormat is not a string");
+
+      alarmMgr.badInput(clientIp, "attrsFormat is not a string");
+      return oe.render(ciP, "");
+    }
+
+    std::string         attrsFormatString = attrsFormat.GetString();
+    NotificationFormat  nFormat           = stringToNotificationFormat(attrsFormatString, true);
+
+    if (nFormat == NGSI_NO_NOTIFICATION_FORMAT)
+    {
+      const char*  details  = "invalid attrsFormat (accepted values: normalized, keyvalues, values)";
```

Comment:
"legacy" is also a valid keyword for attrsFormat (see https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/user/v1_v2_coexistence.md#ngsiv1-notification-with-ngsiv2-subscriptions).